src/ZfcTwig/Twig/StackLoader.php

```diff
@@ -41,18 +41,24 @@ public function getDefaultSuffix()
 
     protected function findTemplate($name)
     {
-        $name = (string) $name;
+        $throw = func_num_args() > 1 ? func_get_arg(1) : true;
+        $name  = (string)$name;
 
         // normalize name
         $name = preg_replace('#/{2,}#', '/', strtr($name, '\\', '/'));
 
         if (isset($this->cache[$name])) {
-            return $this->cache[$name];
```

Comment:
i dont realy get it, why throw an `Exception`, if the catch match?

i think it should be like https://github.com/twigphp/Twig/blob/1.x/lib/Twig/Loader/Filesystem.php#L203-L213